assets/js/_version-selector.js

```diff
@@ -191,10 +191,10 @@ class VersionSelector extends HTMLElement {
 
         frag.querySelector('#selected').textContent = `${PREFIX}${this.getAttribute('selected')}`;
 
-        const pathName = location.pathname.replace(/\/docs(\/((latest|\d+\.\d+)\/?)?)?/, '');
+        const pathName = location.pathname.replace(/\/(docs)?(\/((latest|\d+\.\d+)\/?)?)?/, '');
```

Comment:
@peterzhuamazon Could you verify that this change is correct for the version selector to work for both old and new versions? Should we do it like this?

Reply:
I think we can keep this for now, with or without docs it will strip, my only concern is it will strip if there is another `/<number.number>/` in other part of the url string which will get stripped as well.